docs/Data/Semiring/Boolean.md

```diff
@@ -0,0 +1,28 @@
+## Module Data.Semiring.Boolean
+
+#### `BooleanSemiring`
+
+``` purescript
+newtype BooleanSemiring a
+  = BooleanSemiring a
+```
+
+Provides the semiring for any boolean algebra.
```

Comment:
It might be worth clarifying here that the `Semiring` picks disjunction for `+`, since flipping the operators would give a valid `Semiring` too.